src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,50 @@
+.page {
+  margin: 0;
+}
```

Comment:
it's would be better to move this into `main.scss`, and keep only those styles that related to the calendar in this file